declarations/WebpackOptions.d.ts

```diff
@@ -2929,6 +2929,24 @@ export interface JavascriptParserOptions {
 	 * Enable/disable parsing of magic comments in CommonJs syntax.
 	 */
 	commonjsMagicComments?: boolean;
+	/**
+	 * Specifies global mode for dynamic import.
+	 */
+	dynamicImportMode?:
+		| "sync"
+		| "eager"
+		| "weak"
+		| "async-weak"
+		| "lazy"
+		| "lazy-once";
```

Comment:
sync is not possible for `import()`

and `async-weak` is internal. `weak` will lead to `async-weak` automatically when using `import()`

`lazy-once` is only available for an `import()` with expression.

So this should be `eager | weak | lazy | lazy-once`

And `lazy-once` should be accepted for import() without expression too, and behave as `lazy`

Reply:
Awesome